Gzweb support

kapilPython opened the pull request:
*Issue #, if available:*
Adding Gzweb Support

*Description of changes:*
1. change of all model.sdf files to make it compatible to gzweb, the changes still work with gazebo and ros.
2. Added changes to README to allow use of the models with Gzweb.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

kapilPython committed:
changes to model made to support Gzweb

kapilPython committed:
Gzweb test image added

kapilPython committed:
handling image tag for deploy script of gzweb

kapilPython committed:
To handle Portrait Photos

kapilPython committed:
Update aws_DeskPortraitB_01_visual.DAE

kapilPython committed:
Delete gzweb_aws_house.png

kapilPython committed:
gzweb test image added

kapilPython committed:
Update README.md

cheng-kevin commented:
I've tested this in the [Cloudwatch Sample App](https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch) in ROS Melodic, Gazebo 9.14 on Ubuntu 18.04 and it seems to be working.

cheng-kevin commented:
Also tested in Gzweb 1.4.0,  in ROS Melodic, Gazebo 9.14 on Ubuntu 18.04 and it seems to be working. Node 8.11.3, nodejs 12.19.0, npm 5.6.0. 

Gzweb installation was troublesome.  Needs the exact node versions listed above.  Node-gyp was installed globally

cheng-kevin commented on README.md:
````diff
@@ -25,6 +29,34 @@ export GAZEBO_MODEL_PATH=`pwd`/models
 ```
 
+## Example: Running this world on Gazebo headless and running the UI on Gzweb
+*Tested in ROS Kinetic, Gazebo 7 with node version 10.22.1*
````
Could you include a more dire warning regarding the potential pain of installing Gzweb, outside of these tested versions?  ( I tested in ROS Melodic, Gazebo 9.14 , Node 8.11.3)

kapilPython replied:
For avoiding the pain of installation for gzweb I created a dockerfile which works with ROS kinetic, gazebo 7.16 and node version 10.22.1 the image could be easily pulled from https://hub.docker.com/r/kapildesh/gzweb7.

cheng-kevin replied:
Great, thank you!

kapilPython committed:
Update README.md

corrected gzweb house image title and provided some instructions on handling node-gyp errors.

cheng-kevin reviewed (approved):
Looks good to me
